addon-SV_OptimizedListQueries.xml

```diff
@@ -22,9 +23,15 @@
   <email_templates/>
   <email_template_modifications/>
   <optiongroups>
+    <option option_id="sv_async_queries" edit_format="onoff" data_type="string" can_backup="1">
```

Comment:
This option doesn't actually do anything, as this async support is always enabled by default